### PR TITLE
update docs with splunk.com links

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

File: activemq/SMART_AGENT_MONITOR.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Below is an overview of that monitor.
 
 
 SignalFx's integration with ActiveMQ wraps the [Collectd GenericJMX
-monitor](https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-genericjmx.html)
+monitor](https://docs.splunk.com/Observability/gdi/genericjmx/genericjmx.html)
```
Comment (breedx-splk) [inline activemq/SMART_AGENT_MONITOR.md:22]: Can we lowercase all this `Observability`?

Comment (atoulme) [reply activemq/SMART_AGENT_MONITOR.md:22]: This is how the redirect was set up, I can do that.

Comment (atoulme) [reply activemq/SMART_AGENT_MONITOR.md:22]: Please review https://github.com/signalfx/signalfx-agent/pull/2667 :)

Comment (atoulme) [reply activemq/SMART_AGENT_MONITOR.md]: now fixed in this PR as well.

**Pull request merged**